llvm/test/CodeGen/NVPTX/convert-sm80.ll

```diff
@@ -135,3 +135,24 @@ ret i32 %val
 }
 
 declare i32 @llvm.nvvm.f2tf32.rna(float)
+
+
+define <2 x bfloat> @fold_ff2bf16x2(float %a, float %b) {
+; CHECK-LABEL: fold_ff2bf16x2
```

Comment:
We may as well automate check generation in this test file, too. For vector construction, theorder of inputs is important and we're not capturing it in this file right now.

Reply:
Sounds good, I've updated the test file.